tests/frontend/test_kibana_app/cypress/integration/step-definitions/filters/the-user-add-a-new-filter.and.js

```diff
@@ -0,0 +1,24 @@
+import { And } from 'cypress-cucumber-preprocessor/steps';
+import { clickElement, fillSelectorField, elementIsVisible} from '../../utils/driver';
+
+import {
+  addFilterButton,
+  filterSuggestionList,
+  filterOperatorList,
+  filterParams,
+  saveFilterButton,
+  selectedOperator
+} from '../../pageobjects/filters/filters.page';
+ 
+
+And('The user add a new filter', () => {
```

Comment:
When

Reply:
The reason that this action has to be performed in an AND it's because it's needed that the user navigate previously to a module where the filter function could be called.